### ISSUE TITLE
convnextv2_base timm model conversion fails

### ISSUE THREAD

**Issue by kossyrev-bg** [PINTO0309/onnx2tf#742]
### Issue Type

Others

### OS

Linux

### onnx2tf version number

1.26.8

### onnx version number

1.17.0

### onnxruntime version number

1.21.0

### onnxsim (onnx_simplifier) version number

0.4.36

### tensorflow version number

2.18.0

### Download URL for ONNX

https://drive.google.com/file/d/1IRKaIMXRDfFGurlp08uXaydJRg0yNltp/view?usp=sharing

### Parameter Replacement JSON

```json
{
    "format_version": 1,
    "operations": [
        {
            "op_name": "wa/model/stages/stages.0/blocks/blocks.0/mlp/grn/Mul_1",
            "param_target": "inputs",
            "param_name": "wa/model/stages/stages.0/blocks/blocks.0/mlp/grn/Mul_output_0",
            "pre_process_transpose_perm": [
                0,
                3,
                1,
                2
            ]
        }
    ]
}
```

### Description

Background: I'm trying to deploy a vision model onto a production system at work. I trained a `convnextv2_base` model from the Pytorch Image Models repo `timm`. I modified it to output 2 classes. To simplify things a bit, after training, I created a new model that takes in a uint8 tensor of shape NCHW and does all the `torchvision` preprocessing on it inside the model, so I could port the model to a system that doesn't have torchvision installed. This involves a conversion float32 tensor and normalization. 

I then converted the model to onnx using 
```
dummy_input = torch.randint(0, 255, (1, 3, model_input_shape, model_input_shape), dtype=torch.uint8)
example_inputs = (dummy_input,)
onnx_program = torch.onnx.export(preprocessed_model, example_inputs, "model.onnx", optimize=True, opset_version=12)
```

When attempting to convert the model using `onnx2tf -i model.onnx -b 1 -osd`, I got the following error:
```
Dimensions must be equal, but are 56 and 512 for '{{node tf.math.multiply_29/Mul}} = Mul[T=DT_FLOAT](Placeholder, tf.math.multiply_29/Mul/y)' with input shapes: [?,56,56,512], [1,1,512,1].
```
I understand that it's complaining about the dimension mismatch. What I don't understand is if I scroll up further in the logs, I see:
```
INFO: 41 / 1147
INFO: onnx_op_type: Mul onnx_op_name: wa/model/stages/stages.0/blocks/blocks.0/mlp/grn/Mul_1
INFO:  input_name.1: onnx::Mul_2046 shape: [1, 1, 1, 512] dtype: float32
INFO:  input_name.2: wa/model/stages/stages.0/blocks/blocks.0/mlp/grn/Mul_output_0 shape: [1, 56, 56, 512] dtype: float32
INFO:  output_name.1: wa/model/stages/stages.0/blocks/blocks.0/mlp/grn/Mul_1_output_0 shape: [1, 56, 56, 512] dtype: float32
```
Specifically, I don't understand why we go from `[1,1,1,512]` in the `input_name.1` block to a mismatch where the dims are `[1,1,512,1]`. It seems like some kind of dimensions transposition is happening behind the scenes, but I don't know why.

I did try fixing this using the `replacement.json` method, which worked before failing ~50 ops later. Mainly, I want to understand why there's a transposition happening that doesn't appear to be necessary and if there's anything I can do other than just manually modify them all in a replacement.json.

**Comment by PINTO0309** [comment]
I can't download the ONNX file.

**Comment by kossyrev-bg** [comment]
Gah Box is stupid. @PINTO0309 please try again, I updated the link

**Comment by PINTO0309** [comment]
I found out that there is a bug in tensorflow's `tf.norm()`. 

```
INFO: 21 / 819
INFO: onnx_op_type: ReduceMean onnx_op_name: wa/model/stages/stages.0/blocks/blocks.0/mlp/grn/ReduceMean
INFO:  input_name.1: wa/model/stages/stages.0/blocks/blocks.0/mlp/grn/ReduceL2_output_0 shape: [1, 1, 1, 512] dtype: float32
INFO:  output_name.1: wa/model/stages/stages.0/blocks/blocks.0/mlp/grn/ReduceMean_output_0 shape: [1, 1, 1, 1] dtype: float32
INFO: tf_op_type: reduce_mean
INFO:  input.1.axis0: val: 3 
INFO:  input.2.input_tensor: name: tf.compat.v1.norm_6/norm/transpose_1:0 shape: (None, None, None, None) dtype: <dtype: 'float32'> 
INFO:  input.3.keepdims: val: True 
INFO:  output.1.output: name: tf.math.reduce_mean_4/Mean:0 shape: (None, None, None, 1) dtype: <dtype: 'float32'> 

```

**Comment by PINTO0309** [comment]
Fix: https://github.com/PINTO0309/onnx2tf/releases/tag/1.26.9

```
onnx2tf -i model.onnx
```

**Comment by kossyrev-bg** [comment]
Amazing, thank you so much for the fast help!
